richtext-commonmark/src/commonMain/kotlin/com/halilibo/richtext/markdown/MarkdownParseOptions.kt

```diff
@@ -0,0 +1,16 @@
+package com.halilibo.richtext.markdown
+
+/**
+ * Allows configuration of the Markdown parser
+ *
+ * @param autolink Detect plain text links and turn them into Markdown links.
+ */
+public data class MarkdownParseOptions(
+  val autolink: Boolean
+) {
+  public companion object {
+    public fun getDefaults(): MarkdownParseOptions = MarkdownParseOptions(
```

Comment:
nit; you can make this a `Default` val instead of re-allocating with every function call

Reply:
Done!